docs/data-builder-dummy-data.md

```diff
@@ -0,0 +1,39 @@
+# Supplying a dummy dataset
+
+---8<-- 'includes/data-builder-danger-header.md'
+
+Because OpenSAFELY doesn't allow direct access to individual patient
+records, researchers must use a **dummy dataset** for developing their analytic
+code on their own computer.
+
+A dummy dataset needs to be similar enough to a real dataset to allow
+the analysis scripts to run successfully, but it must be randomly (or
+psuedo-randomly) generated and not contain data relating to real
+patients.
+
+In future, Data Builder will be able to generate suitable dummy data
+from a dataset definition, but for now you must generate your own dummy
+data file and use the `--dummy-data-file` argument to tell Data Builder
+where it is. When running on your own computer (or in the automated
+tests), Data Builder will copy this file to the same place it will put
```

Comment:
suggestion: For consistency, you might want to refer to this situation in the same way as suggested in [@iaindillingham's comment earlier](https://github.com/opensafely/documentation/pull/635/files#r821938781). That is: referring to "when running outside of a secure EHR vendor environment" or similar.

Reply:
thought: "same place" here is possibly ambiguous. For example, on my computer, I cannot access a vendor's file system. Do you mean something along the lines of "the same relative file path"?